packages/js-dpp/lib/errors/consensus/codes.js

```diff
@@ -123,6 +124,7 @@ const codes = {
   1040: InvalidIdentityPublicKeyDataError,
   1041: InvalidInstantAssetLockProofError,
   1042: InvalidInstantAssetLockProofSignatureError,
+  1046: MissingMasterKeyError,
```

Comment:
It seems we don't use just "key" term in DPP. MissingMasterPublicKeyError?

Reply:
Renamed